bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/BookieRequestProcessor.java

```diff
@@ -369,6 +378,40 @@ private void processAddRequestV3(final BookkeeperProtocol.Request r, final Chann
         }
     }
 
+    private void processForceLedgerRequestV3(final BookkeeperProtocol.Request r, final Channel c) {
+        ForceLedgerProcessorV3 forceLedger = new ForceLedgerProcessorV3(r, c, this);
+
+        final OrderedExecutor threadPool;
+        if (RequestUtils.isHighPriority(r)) {
```

Comment:
@sijie I don't know if it is worth using this threadpool for forceLedger, I copied this from addEntry, for consistency

Reply:
I think it is okay to be consistent